### PR TITLE
Add 10.15 and 10.14 to package requirements for Bartender and 1Password

### PR DESCRIPTION
**Pull request by krispayne**
Recently set up a new 10.15 machine, Bartender and 1Password package requirements are out of date.

### CODE REVIEW

**Comment by krispayne** [comment]
bump

File: Bartender/Bartender.jss.recipe [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		<key>NAME</key>
 		<string>Bartender</string>
 		<key>OS_REQUIREMENTS</key>
-		<string>10.13.x,10.12.x,10.11.x,10.10.x</string>
+		<string>10.15.x,10.14.x,10.13.x,10.12.x,10.11.x,10.10.x</string>
```
Comment (homebysix) [inline Bartender/Bartender.jss.recipe]: According to the Bartender website, the latest version requires 10.12+.
image

File: 1Password/1Password.jss.recipe [hunk 2]
```diff
@@ -17,7 +17,7 @@
 		<key>NAME</key>
 		<string>1Password</string>
 		<key>OS_REQUIREMENTS</key>
-		<string>10.14.x,10.13.x,10.12.6</string>
+		<string>10.15.x,10.14.x,10.13.x,10.12.6</string>
```
Comment (homebysix) [inline 1Password/1Password.jss.recipe]: According to the 1Password website, the latest version requires 10.13+.
image

**Comment by krispayne** [comment]
Good catches! I'll update the bottom ends as well

**Comment by homebysix** [comment]
Looks good, thank you!

**Pull request merged**